### ISSUE TITLE
Comments are outdented incorrectly in script blocks

### ISSUE THREAD

**Issue by garraflavatra** [sveltejs/eslint-plugin-svelte#557]
### Before You File a Bug Report Please Confirm You Have Done The Following...

- [X] I have tried restarting my IDE and the issue persists.
- [X] I have updated to the latest version of the packages.

### What version of ESLint are you using?

8.46.0

### What version of `eslint-plugin-svelte` are you using?

2.32.4

### What did you do?

I linted the following outdented code using the enclosed ESLint configuration.

```svelte
<script>

function demo1() {
// This should be indented with 4 spaces
console.log('ok');
}

function demo2() {
// This should be indented with 4 spaces
}

function demo3() {
// This should be indented with 4 spaces
// Another line
console.log('ok');
}

function demo4() {
// This should be indented with 4 spaces
// Another line
}

demo1();
demo2();
demo3();
demo4();

</script>
```

```json
{
  "parserOptions": {
    "ecmaVersion": 2020,
    "sourceType": "module"
  },
  "env": {
    "browser": true,
    "node": true,
    "es6": true
  },
  "ignorePatterns": [
    "/expected.svelte",
    "/original.svelte"
  ],
  "extends": [
    "eslint:recommended",
    "plugin:svelte/recommended"
  ],
  "plugins": [
    "svelte"
  ],
  "rules": {
    "svelte/indent": "error"
  }
}
```

### What did you expect to happen?

I expected ESLint to indent the code correctly; in my opinion, that is:

```svelte
<script>

  function demo1() {
    // This should be indented with 4 spaces
    console.log('ok');
  }

  function demo2() {
    // This should be indented with 4 spaces
  }

  function demo3() {
    // This should be indented with 4 spaces
    // Another line
    console.log('ok');
  }

  function demo4() {
    // This should be indented with 4 spaces
    // Another line
  }

  demo1();
  demo2();
  demo3();
  demo4();

</script>
```

### What actually happened?

Fixing it by running `eslint --fix` changed it to the following:

```svelte
<script>

  function demo1() {
    // This should be indented with 4 spaces
    console.log('ok');
  }

  function demo2() {
  // This should be indented with 4 spaces
  }

  function demo3() {
    // This should be indented with 4 spaces
    // Another line
    console.log('ok');
  }

  function demo4() {
  // This should be indented with 4 spaces
    // Another line
  }

  demo1();
  demo2();
  demo3();
  demo4();

</script>
```

### Link to **GitHub Repo** with Minimal Reproducible Example

Repository: https://github.com/garraflavatra/svelte-eslint-comment-repro.

The original code is in `original.svelte`, the expected result is in `expected.svelte`, and the actual output is in `actual.svelte`. The `.eslintrc.json` file contains the used (minimal) configuration.

### Additional comments

I tried looking for similar issues before creating this one, but found none.

**Comment by thenbe** [comment]
In objects, this bug only happens on the *first* commented out key.

Here, it will complain only about the `c` key.
## Repro

```svelte
<script>
  /* eslint svelte/indent: "error" */
  const obj1 = {
    a: '',
    b: '',
    // c: '',
    // d: '',
    // e: '',
  }
</script>
```

- EXPECTED: 'svelte/indent' should not complain about indent level.
- ACTUAL: 'svelte/indent' has one complaint. It complains about the indent level for line number 6 (the line containing the `c` key). 'svelte/indent' wants line number 6 to be indented one level less than line number 5 and one level less than line number 7.

---

I tried creating a [playground](https://sveltejs.github.io/eslint-plugin-svelte/playground/#eJxVzTEKgDAQRNGrDNvYCEHLoJ7ERpMtlLCRJNqE3F1EEe3+m2YyGW+ZNHXRhGVLwyiA8RIT/Lw26JGvBZg0qqq+e/60UjB/2j/5ZRmlU88N1RR2x5F0pniwS6wWsSyJdFvKCeUFKWk=) link but the indent rule doesn't work at all there (I disabled all rules except svelte/indent). But if you paste my snippet into the rule's example [page](https://sveltejs.github.io/eslint-plugin-svelte/rules/indent/) you can reproduce the issue.
